Lusas_Adapter/Convert/ToBHoM/Loads/ToLoadcase.cs

```diff
@@ -35,13 +35,14 @@ public static partial class Convert
 
         public static Loadcase ToLoadcase(this IFLoadcase lusasLoadcase)
         {
+            long adapterNameId = lusasLoadcase.getID();
+
             Loadcase loadcase = new Loadcase
             {
                 Name = GetName(lusasLoadcase),
-                Number = lusasLoadcase.getID()
+                Number = (int)adapterNameId,
```

Comment:
Same as above comment ^

Reply:
Added with warning message as this will impact combinations that are pulled.